atomicapp/nulecule/base.py

```diff
@@ -403,3 +403,24 @@ def render_artifact(self, path, context):
             self.basepath + ('' if self.basepath.endswith('/') else '/'),
             1)[1]
         return render_path
+
+    def _get_artifact_paths_for_path(self, path):
+        """
+        Get artifact paths for a local filesystem path.
+
+        Args:
+            path (str): Local path
+
+        Returns:
+            list: A list of artifact paths
+        """
+        artifact_paths = []
+        if os.path.isfile(path):
+            artifact_paths.append(path)
+        elif os.path.isdir(path):
+            for dir_child in os.listdir(path):
```

Comment:
The `path` given by user is a path of directory having a set of k8s manifests in it. Do we need to `os.listdir` and traverse in sub directories as well?
@aweiteka : am I missing something here?